src/ServiceManager.php

```diff
@@ -259,6 +259,9 @@ public function build($name, ?array $options = null)
 
     /**
      * {@inheritDoc}
+     *
+     * @param string|class-string $name
```

Comment:
`string` contains `class-string` :thinking:

Reply:
As documentation, this reads to me like - "I might get a regular string literal or a class-string" - is that not an improvement on plain old string?

Reply:
No strong feelings, but other tools do indeed report this as a redundant type union